model/layout.md

```diff
@@ -71,6 +72,24 @@ When layout direction is `"column"`:
 
 **Kind**: instance property of [<code>Layout</code>](#Layout)
 
+<a name="Layout+distribution"></a>
+### layout.distribution : <code>String</code>
+Determines how the children are distributed inside the layer.
+
+When layout direction is `"row"`:
+- `"center"`: 
+- `"max"`: 
+- `"min"`: 
+- `"space-between"`: 
```

Comment:
No need for colon at the end.

Reply:
I'll add explanations, that's why colon is there 😬